static/js/components/widgets/SortableSelect.tsx

```diff
@@ -101,6 +110,7 @@ export default function SortableSelect(props: Props) {
           options={options}
           loadOptions={loadOptions}
           defaultOptions={defaultOptions}
+          {...(isOptionDisabled ? { isOptionDisabled } : {})}
```

Comment:
could also just do `isOptionDisabled={isOptionDisabled}` since the types are the same (`isOptionDisabled?: (option: Option) => boolean`)

Reply:
sometimes I wish there was a JSX syntax like the object literal property shorthand (`let foo = "bar"; let obj = { foo }`)

Reply:
👍 I was trying to distinguish between `props = { isOptionDisabled: undefined }` and `props = { }`. But that's probably overly paranoid and nice to keep the code simpler.

Reply:
My thought is that typescript gives us a lot of assurance that if a prop is `undefined` (vs not set) we still won't end up trying to call it, so I think it's fine to just pass the value down